src/hardware/sblaster.cpp

```diff
@@ -2849,23 +2945,29 @@ bool SB_GetAddress(uint16_t &sbaddr, uint8_t &sbirq, uint8_t &sbdma)
 	return (sbaddr != 0 && sbirq != 0 && sbdma != 0);
 }
 
-static void sblaster_mixer_callback([[maybe_unused]]const int requested_frames)
+static void sblaster_mixer_callback(const int requested_frames)
```

Comment:
So yeah, I don't think this function does anything special now.  I wrote this to handle the `AudioVector` class but now that it's gone, you should be able to delete this entire thing and replace the callback with `MIXER_PullFromQueueCallback`

Reply:
Thanks for all the help with this @weirddan455; I've dropped the `sblaster_mixer_callback` call and switch to the `MIXER_PullFromQueueCallback` pattern and it's working nicely!
I especially like how this function adjusts the queue size on the fly depending on if we're fastwording or not! :1st_place_medal:

Reply:
This is transition is a separate commit (and acts as a nice "step" in the evolution; both the prior and later compile and run)